test/catalog/catalog_test.cpp

```diff
@@ -120,6 +120,30 @@ TEST_F(CatalogTests, CreatingTable) {
   txn_manager.CommitTransaction(txn);
 }
 
+TEST_F(CatalogTests, TestingCatalogCache) {
+  auto &txn_manager = concurrency::TransactionManagerFactory::GetInstance();
+  auto txn = txn_manager.BeginTransaction();
+
+  auto catalog = catalog::Catalog::GetInstance();
+  auto catalog_db_object = catalog->GetDatabaseObject(CATALOG_DATABASE_OID, txn);
+  auto catalog_table_objects = catalog_db_object->GetTableObjects();
+  EXPECT_NE(0, catalog_table_objects.size());
+
+  auto user_db_object = catalog->GetDatabaseObject("emp_db", txn);
+  auto user_database = storage::StorageManager::GetInstance()
+                          ->GetDatabaseWithOid(user_db_object->GetDatabaseOid());
+
+  // check expected table object is acquired
+  for (oid_t table_idx = 0; table_idx < user_database->GetTableCount(); table_idx++) {
```

Comment:
Do we know what tables we have at this point? Can we also verify the number of tables or their specific names?

Reply:
It just checks that the cache of table catalog belongs to a correct database about #1373. The information about creating table might be checked in previous test `CreatingTable`.